packages/generator-electrode/generators/app/templates/src/client/components/demo-dynamic-import.jsx

```diff
@@ -13,7 +13,7 @@ const Fallback = () => (
   </div>
 );
 
-let Demo = loadable(() => import(/* webpackChunkName: "fake" */ "./demo-fake"), {
+let Demo = loadable(() => import("./demo-fake"), {
```

Comment:
what's this change for?

Reply:
remove the chunk for this component,  and make fewer generated js files

Reply:
component name changed and webpackChunkName brought back